clients/web/src/components/console/log-event/field.tsx

```diff
@@ -42,6 +28,27 @@ export function Field(props: { field: IField }) {
       >
         {strVal()}
       </span>
+      <span class="flex flex-row absolute bg-gray-950/50 right-[-0.5rem] top-[-1rem] z-51 p-1 opacity-45 group-hover:opacity-100">
```

Comment:
You don't need to use custom values for these props.

- `right-[-0.5rem]` is the same as `-right-2`
- `top-[-1rem]` is the same as `-top-4`

I'd suggest changing those values slightly to avoid some overlapping though.
 
![Screenshot 2024-06-11 at 17 50 32](https://github.com/crabnebula-dev/devtools/assets/141120083/fa92a2a8-a3cb-4ec1-bdcd-07eb1f75a778)

I tested slightly, and I think `bg-gray-950/75  -right-5 -top-2` is a bit easier to read. (I increased the opacity to add a bit more contrast too).

---

`z-51` is not a valid Tailwind class, I think this was intended to be `z-50`, but probably this can get by unset or with a `z-40` so the tooltip content can be `z-50`